packages/apps-config/src/ui/logos/index.ts

```diff
@@ -359,13 +361,15 @@ export const specLogos: Record<string, unknown> = [
 export const namedLogos: Record<string, unknown> = {
   acala: nodeAcala,
   alexander: nodePolkadot,
+  altair: chainAltair,
   basilisk: nodeBasilisk,
   beast: nodeBeast,
   bifrost: nodeBifrost,
   bitcountry: nodeBitCountry,
   canvas: nodeCanvas,
   centrifuge: nodeCentrifuge,
   chainx: nodeChainx,
+  charcoal: chainAltair,
```

Comment:
Can we default charcoal to the `nodeCentrifuge` logo above instead of Altair one?